internal/pkg/cli/deploy/workload.go

```diff
@@ -170,6 +172,14 @@ type WorkloadDeployerInput struct {
 	customResources customResourcesFunc
 }
 
+// RuntimeImage is the configuration of image digest and tags of an ECR image.
+type RuntimeImage struct {
+	Digest         string
+	CustomTag      string
+	GitShortCommit string
+	UUID           string
```

Comment:
nit: to highlight that all of these are tags, what do you think of adding the suffix:
```suggestion
	CustomTag      string
	GitShortCommitTag string
	UUIDTag           string  // If not specified, a random UUID is generated.
```